scilpy/tracking/seed.py

```diff
@@ -73,18 +73,21 @@ def get_next_pos(self, random_generator, indices, which_seed):
             return []
 
         # Voxel selection from the seeding mask
-        ind = which_seed % len_seeds
+        ind = (which_seed // self.n_repeats) % len_seeds
         x, y, z = self.seeds_vox[indices[ind]]
 
-        if self.randomize_positions:
+        if which_seed % self.n_repeats == 0 or self.previous_offset is None:
```

Comment:
Could you modify the get_next_n_seeds too please? (New line 136)

Reply:
This one is tougher, but I think I'm getting somewhere